engine/src/main/resources/templates/Rbac/platformGroupToCustomEmailBodyV2.html

```diff
@@ -0,0 +1,18 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    User Access - Console
+{/content-title}
+{#content-title-section1}
+    User Access platform group Notification
+{/content-title-section1}
+{#content-body-section1}
+    <p>
+        Platform default group is modified by {action.events[0].payload.username} and the Red Hat will not be responsible for managing it from now on.
```

Comment:
There's a typo in the old sentence:
```suggestion
        Platform default group is modified by {action.events[0].payload.username} and Red Hat will not be responsible for managing it from now on.
```